packages/create-sitecore-jss/src/templates/react/src/components/Styleguide-Multilingual/index.js

```diff
@@ -21,10 +21,8 @@ const StyleguideMultilingual = (props) => (
       <Link to="/en/styleguide">Show in English</Link>
       <br />
       <% if (language)  { -%>
-        <><Link to="/<%- language %>/styleguide">Show in < />%- language %></Link>< /></> % } -%>
-
+       <Link to="/<%- language %>/styleguide">Show in <%- language %></Link>
```

Comment:
need closing ejs bracket
```suggestion
       <Link to="/<%- language %>/styleguide">Show in <%- language %></Link>
      <% } -%>
```